CMakeLists.txt

```diff
@@ -66,6 +66,8 @@ else()
     endif()
 endif()
 
+find_package(VulkanUtilityLibraries CONFIG REQUIRED QUIET)
```

Comment:
```suggestion
find_package(VulkanUtilityLibraries CONFIG REQUIRED)
```

Reply:
I don't think it's a good practice to make a "request change" with no explanation on the matter.

This said, I changed it.